### PR TITLE
feat(icons): add initial Icons package

### PR DESCRIPTION
**Pull request by andresin87**
## feat(icons): add initial package

**TASK**:  #366

### Description, Motivation and Context

When installing the dependencies, the icons package will parse all svg from assets and create its proper tsx. It will also index all of them and place them on the proper stories section.

All svgs from assets will be
- optimized: reduces the commas to 4 max
- pathified: converts all simple shapes to path
- normalized
	- remove class
	- remove width
	- remove height
	- set fill 'currentcolor'
	- set stroke 'curretncolor'
- a11y title label text added
- componentized: converted to tsx
- prettified
- tested

### Types of changes

- [x] ✨ New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: packages/components/icons/src/icons/Check.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export const Check = () => (
+  <svg
+    viewBox="0 0 24 24"
+    xmlns="http://www.w3.org/2000/svg"
+    data-title="check"
+    fill="currentColor"
+    stroke="currentColor"
+  >
+    <title>check</title>
```
Comment (arnau-rius) [inline packages/components/icons/src/icons/Check.tsx:9]: 👀

**Review by arnau-rius — COMMENTED** [review]
Is it ok to have to check svg's? 
1. packages/components/icons/src/icons/Check.tsx
2. packages/components/icons/assets/check.svg

File: packages/components/icons/src/icons/Check.tsx (hunk 1 above)
Comment (andresin87) [reply packages/components/icons/src/icons/Check.tsx:9]: this is 4 a11y purposes

Comment (andresin87) [reply packages/components/icons/src/icons/Check.tsx:9]: https://testing-library.com/docs/queries/bytitle/

**Review by arnau-rius — APPROVED** [review]

File: packages/components/icons/src/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { render, screen } from '@testing-library/react'
+import { describe, expect, it } from 'vitest'
+
+import * as Icons from './index'
+
+describe('Icons', () => {
+  describe('', () => {
```
[the remark this answers is not in the document]
Comment (Powerplex) [reply packages/components/icons/src/index.test.tsx:7]: 👁👄👁

**Comment by andresin87** [comment]
> Is it ok to have to check svg's?
> 
> 1. packages/components/icons/src/icons/Check.tsx
> 2. packages/components/icons/assets/check.svg

it looks good to easy track where component changes come from

**Comment by andresin87** [comment]
> Maybe we could add a README or something similar to explain the process to add icons to this package in the future ? wdyt ?
DONE. thanks. good idea.
https://github.com/adevinta/spark/pull/458/commits/29dc029e005e9ad283a2037484dc23b06df48ba5

**Pull request merged**